docs/user_guide/application.md

```diff
@@ -21,32 +20,32 @@ class MyApplication(JupyterApp):
 Register an event schema with the logger.
 
 ```python
-        schema = """
-        $id: http://myapplication.org/my-method
-        version: 1
-        title: My Method Executed
-        description: My method was executed one time.
-        properties:
-        msg:
-            title: Message
-            type: string
-        """
-
-        self.eventlogger.register_event_schema(
-            schema=schema
-        )
+schema = """
+$id: http://myapplication.org/my-method
+version: 1
+title: My Method Executed
+description: My method was executed one time.
+properties:
+msg:
+    title: Message
+    type: string
+"""
+
+self.eventlogger.register_event_schema(schema=schema)
 ```
 
 Call `.emit(...)` within the application to emit an instance of the event.
 
 ```python
-    def my_method(self):
-        # Do something
-        ...
-        # Emit event telling listeners that this event happened.
-        self.eventlogger.emit(schema_id="myapplication.org/my-method", data={"msg": "Hello, world!"})
-        # Do something else...
-        ...
```

Comment:
Same here. This indent is intentional.

Reply:
Hmm, this was auto-formatted by blacken-docs, maybe we need to skip this file.

Reply:
Fixed by skipping